content/master/concepts/patch-and-transform.md

```diff
@@ -1562,6 +1567,9 @@ converts the input based on one of the following conversion types:
 * `ToSha1` - Create a SHA-1 hash of the input string.
 * `ToSha256` - Create a SHA-256 hash of the input string.
 * `ToSha512` - Create a SHA-512 hash of the input string.
+* `ToSha512` - Create a SHA-512 hash of the input string.
+* `ToAdler32` - Create an Adler32 hash of the input string.
+* `ToSha512` - Create a SHA-512 hash of the input string. 
 * `ToAdler32` - Create an Adler32 hash of the input string.
```

Comment:
don't know if it was about the sorting or missing transforms, but adler32 and sha512 were already present:
```suggestion
* `ToSha1` - Create a SHA-1 hash of the input string.
* `ToSha256` - Create a SHA-256 hash of the input string.
* `ToSha512` - Create a SHA-512 hash of the input string.
* `ToAdler32` - Create an Adler32 hash of the input string.
```

Reply:
copy pasta mistake. good catch!